compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -437,6 +441,26 @@ private void defineTypeDef(DataInputStream dataInStream) throws IOException {
         }
     }
 
+    private void setInvokableTypeSymbol(BInvokableType invokableType) {
+        BInvokableTypeSymbol tsymbol = (BInvokableTypeSymbol) invokableType.tsymbol;
```

Comment:
WDYT of changing the variable name to camel case here? 
```suggestion
        BInvokableTypeSymbol tSymbol = (BInvokableTypeSymbol) invokableType.tsymbol;
```

Reply:
@dulvinw Although I  agree with your suggestion,  this ('tsymbol`) seems to be the practice followed by majority of the areas in compiler frontend. Hence, i think we can keep it as it is now.  
This is something we can do in a separate refactoring  (change `tsymbol` everywhere) PR.